opensrp-chw-anc/src/main/java/org/smartregister/chw/anc/util/JsonFormUtils.java

```diff
@@ -218,6 +225,32 @@ public static String getValue(JSONObject jsonObject, String key) {
         return "";
     }
 
+    /**
+     * Returns an Array value from JSON form field
+     *
+     * @param formJsonObject native forms jsonObject
+     * @param step       JSON Form step with the field
+     * @param key        field object key
+     * @return Array value
+     */
+    public static JSONArray getValueArray(JSONObject formJsonObject, String step, String key) {
+        try {
+            JSONArray fieldsArray = formJsonObject.getJSONObject(step).getJSONArray(JsonFormConstants.FIELDS);
```

Comment:
Check for null.

Reply:
This is handled in the try-catch that handles parsing the JSONObject and JSONArray